src/schedule/zephyr_ll.c

```diff
@@ -36,6 +36,16 @@ struct zephyr_ll_pdata {
 	struct k_sem sem;
 };
 
+static void zephyr_ll_lock(struct zephyr_ll *sch, uint32_t *flags)
```

Comment:
Can we call the Zephyr API directly ?

Reply:
@lgirdwood don't think so, not yet. Zephyr spinlocks use `struct k_spinlock` and XTOS uses `spinlock_t`. But this spinlock is removed in a follow up commit anyway, it's just abstracted here for convenience, for easier replacement and removal.